resources/views/campaigns/_users.blade.php

```diff
@@ -81,15 +84,17 @@
                                         <span class="sr-only">{{ __('crud.actions.actions') }}</span>
                                     </a>
                                     <ul class="dropdown-menu dropdown-menu-right" role="menu">
-                                        @can('switch', $relation)
-                                            <li>
-                                                <a href="{{ route('identity.switch', $relation) }}" title="{{ __('campaigns.members.helpers.switch') }}" data-toggle="tooltip" class="switch-user">
-                                                    <i class="fa-solid fa-sign-in-alt" aria-hidden="true"></i>
-                                                    {{ __('campaigns.members.actions.switch') }}
-                                                </a>
-                                            </li>
-                                            <li class="divider"></li>
-                                        @endcan
+                                        @if (!$relation->user->isBanned())
+                                            @can('switch', $relation)
```

Comment:
Instead of doing an if here (which means forgetting to add it to the controller), we can add the check directly in the CampaignMemberPolicy and revert back the dropdown option